runbot_travis2docker/models/runbot_branch.py

```diff
@@ -76,35 +71,44 @@ def cron_weblate(self):
                         continue
                     if project['name'] != branch.name_weblate:
                         continue
-                    response = session.get('%s/components/%s/%s/changes/'
-                                           % (url, project['slug'],
-                                              component['slug']))
-                    response.raise_for_status()
-                    changes = response.json()
-                    if not changes['results']:
+                    has_build = self.env['runbot.build'].search(
+                        [('branch_id', '=', branch.id),
+                         ('state', 'in', ('pending', 'running', 'testing')),
+                         ('name', '=', component['branch']),
+                         ('uses_weblate', '=', True)])
+                    if has_build:
                         continue
-                    change = None
-                    for record in changes['results']:
-                        if record['action'] == 17:
-                            change = record
-                            break
-                    if not change:
+                    remote = 'wl-%s' % project['slug']
+                    url_repo = (branch.repo_id.weblate_url.replace('api',
+                                                                   'git') +
+                                '/' + project['slug'] + '/' +
+                                component['slug'])
+                    try:
+                        subprocess.check_output(cmd + ['remote', 'add', remote,
+                                                url_repo])
+                    except subprocess.CalledProcessError:
+                        pass
+                    subprocess.check_output(cmd + ['fetch', remote])
+                    diff = subprocess.check_output(
+                        cmd + ['diff',
+                               '%(branch)s..%(remote)s/%(branch)s'
```

Comment:
@moylop260 
I try `git diff origin/10.0..wl-github_com_JesusZapata_openacademy-project_10_0_/10.0` but this show me on error `fatal: ambiguous argument`

![image](https://cloud.githubusercontent.com/assets/1387970/26653912/b9715fb0-4622-11e7-84ae-4fb24c4896b5.png)